packages/manager/modules/pci/src/projects/onboarding/onboarding.module.js

```diff
@@ -6,6 +6,8 @@ import 'angular-translate';
 import '@ovh-ux/ui-kit';
 
 import routing from './onboarding.routing';
+import orderCart from '../order-cart.service'; // TODO remove it
```

Comment:
Why removing it if you're currently using it ?

Reply:
I aligned my implementation on the implementation of the new project creation script `packages/manager/modules/pci/src/projects/new/module.js`.
So, I left this comment to keep track of the intent of the developer who wanted to refactor it at some point.